problems/removeEvenStrings.js

```diff
@@ -6,7 +6,14 @@
 * @returns {string[]} - Returns the strings in arr that have an odd number of characters
 */
 
-function removeEvenStrings() {
+function removeEvenStrings(arr) {
+    newArr = []
+    arr.filter((str) => {
+        if(str.length % 2 === 1){
+            newArr.push(str)
```

Comment:
Incorrect use of filter. Don't give filter side effects by pushing into an array. Instead use filter to return the new array itself by having a truthy return in the callback.